collections/_article/introducing-the-improved-ufbx-importer-in-godot-4-3.md

```diff
@@ -0,0 +1,57 @@
+
+* [#89629 (Import rest pose as RESET animation)](https://github.com/godotengine/godot/pull/89629): This enables creating a RESET track to restore the skeleton to its imported pose. 
+* [#88824 (Retargeting option to use a template for silhouette)](https://github.com/godotengine/godot/pull/88824): This feature can be used to reference the RESET animation of a known good (T-pose) reference file. Having a separate reference pose .fbx is a common practice in some animation sets.
+* [#88821 (Allow preserving the initial bone pose in rest fixer)](https://github.com/godotengine/godot/pull/88821): This allows for importing a retargeted skeleton while keeping it in its initial pose.
+* [#88819 (Add new scene import option to import as Skeleton)](https://github.com/godotengine/godot/pull/88819): This solves cases especially common in .fbx files without any meshes, where animated skeletons import as a series of nodes.
+* Improvements to bone detection [#90050 (Add tips detection to auto mapping in bone mapper)](https://github.com/godotengine/godot/pull/90050): Improves support for Mixamo and ReadyPlayerMe finger bone conventions.
+* Several other bugfixes ([#90019](https://github.com/godotengine/godot/pull/90019), [#90064](https://github.com/godotengine/godot/pull/90064), [#90065](https://github.com/godotengine/godot/pull/90065), [#91641](https://github.com/godotengine/godot/pull/91641), [#92012](https://github.com/godotengine/godot/pull/92012)) to retargeting and animation import.
+
+# Issues and testing
+
+Samuli has spent exceptional time and effort testing the ufbx library on tens of thousands of .fbx test cases from a variety of sources, and we are proud to consider the new ufbx importer feature-complete. This level of rigorous design and testing is a stark contrast to previous .fbx importer rewrites some of you may recall from Godot 3.
+
+We expect that bugs with .fbx import are going to be relatively uncommon, but if you encounter something incorrect or broken, please [report the issues you find](/godotengine/godot/issues) and include the .fbx files which reproduce the issue or reach out to us with instructions.
+
+# Conclusion
+
+The ufbx importer is a major step in improving the accuracy and ease of use of Godot’s 3D importer. It is but one of many improvements, and there is more to come in future 4.x releases.
+
+There are sure to be many applications for the native .fbx importer. Our hope is it will improve interoperability with a wide variety of animation tools and libraries out in the wild. For example, I and the V-Sekai team has integrated ufbx into Unidot Importer to allow for interoperability with asset packs in .fbx format.
```

Comment:
```suggestion
There are sure to be many applications for the native .fbx importer. Our hope is it will improve interoperability with a wide variety of animation tools and libraries out in the wild. For example, I and the V-Sekai team have integrated ufbx into Unidot Importer to allow for interoperability with asset packs in .fbx format.
```